pkg/cli/cmd/bicep/generatekubernetesmanifest/testdata/aws/aws.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: radapp.io/v1alpha3
+kind: DeploymentTemplate
+metadata:
+  name: aws.bicep
+  namespace: radius-system
+spec:
+  parameters: {}
+  providerConfig: |-
+    {
+      "radius": {
+        "type": "radius",
```

Comment:
same question as above for the provider types. are these the correct type values?

Reply:
Comparison with the `rad deploy` code is the right way to answer this question.